services/web/server/tests/unit/with_dbs/01/test_storage.py

```diff
@@ -255,14 +256,17 @@ async def test_get_files_metadata_dataset(
         (UserRole.TESTER, web.HTTPOk),
     ],
 )
-async def test_storage_file_meta(client, storage_server, logged_user, expected):
+async def test_storage_file_meta(
+    client, storage_server, logged_user, expected, faker: Faker
+):
     # tests redirect of path with quotes in path
-    file_id = "a/b/c/d/e/dat"
-    url = "/v0/storage/locations/0/files/{}/metadata".format(quote(file_id, safe=""))
+    file_id = f"{faker.uuid4()}/{faker.uuid4()}/a/b/c/d/e/dat"
```

Comment:
was this failing before? or is this due to the openapi changes?

Reply:
This was not failing before, with the old openapi. Now the new specs validate using the the proper type and specs.